app/controllers/admin/custom_organisations_controller.rb

```diff
@@ -9,11 +9,17 @@ def create
     @custom_organisation = CustomOrganisationName.new(custom_org_params)
 
     if @custom_organisation.save
-      flash[:notice] = "Custom organisation has been successfully added"
+      flash.now[:notice] = "Custom organisation has been successfully added"
     end
     render :index
   end
 
+  def destroy
+    custom_org = CustomOrganisationName.find(params.fetch(:id))
+    custom_org.destroy
```

Comment:
Not vital because it's just the super admins who see this, but the convention is to only redirect if it was successful:

```
if custom_org.destroy
    redirect_to admin_custom_organisations_path, notice: "Successfully removed #{custom_org.name}"
end
```

Will prevent confusing where it says it deleted the org but that didn't really happen.